src/ai/backend/test/contexts/tester.py

```diff
@@ -4,4 +4,6 @@
 
 
 class TestIDContext(BaseTestContext[uuid.UUID]):
-    pass
+    @classmethod
+    def get_name(cls) -> str:
+        return "test_id"
```

Comment:
I don't particularly prefer putting a `get_` prefix on values for getters. How about leaving it as a property instead?

Reply:
Ah, because it's a classmethod, it probably won't work simultaneously... How about just changing the name to "name"?

Reply:
To use `classproperty` in Python, we need an additional fixture.
See https://stackoverflow.com/questions/128573/using-property-on-classmethods
If you have a preferred implementation, please let me know and I will update PR.

Reply:
https://github.com/lablup/backend.ai/pull/4661#discussion_r2144024428
Ok, I will reflect them after you commit